### PR TITLE
1.21

### PR DESCRIPTION
**Pull request by orifu**
- ~~port to 1.21 pre release 1~~
  - ~~match if you want pre 2 >:(~~
  - port to 1.21 rc 1
- ~~add /ramel command to access config (no modmenu 1.21 yet)~~
  - ~~adds fapi dep~~

### CODE REVIEW

**Review by eristhea — CHANGES_REQUESTED** [review]
lovely work! except the command. I think it's overrated it's like a keypress and 2 clicks to get there without

File: build.gradle [hunk 1]
```diff
@@ -11,6 +11,8 @@ repositories {
     maven { url = "https://maven.quiltmc.org/repository/release/" }
     maven { url = "https://maven.terraformersmc.com/releases/" }
     maven { url = "https://repo.sleeping.town/" }
+
+    flatDir { dirs 'lib' }
```
Comment (eristhea) [inline build.gradle]: what does this do?

Comment (orifu) [reply build.gradle]: loads jar libraries from the lib folder. temporary until modmenu 1.21 goes on their maven

Comment (eristhea) [reply build.gradle]: very cool i will remember this trick

**Review by eristhea — APPROVED** [review]

**Pull request merged**